pom.xml

```diff
@@ -740,7 +740,6 @@
                         <jvmVersion>1.8</jvmVersion>
                         <jvmOptions>
                             <string>-XstartOnFirstThread</string>
-                            <string>-Dsalesforce.config.dir=Contents/Resources/conf</string>
```

Comment:
Previously, in Mac profile we were ALWAYS passing in "Contents/Resources/conf" as "salesforce.config.dir" and this path is world accessible. We need to dynamically set this directory during the runtime, not during packaging.